sources/packages/backend/apps/api/src/route-controllers/application-exception/_tests_/application-exception-helper.ts

```diff
@@ -41,13 +50,17 @@ export async function saveFakeApplicationWithApplicationException(
     creator: student.user,
     assessedBy,
   });
-  applicationException.exceptionStatus = applicationExceptionStatus;
+  applicationException.exceptionStatus = options?.applicationExceptionStatus;
```

Comment:
If the applicationExceptionStatus is not provided in the options, it will set the exception status to undefined right?

Reply:
By default, it is pending from line 49, but you are correct, since options are optional, it will rewrite it. updated